ui/browser/store/store.js

```diff
@@ -13,6 +13,14 @@ specific language governing permissions and limitations under the License.
 import { applyMiddleware, createStore } from 'redux';
 import createLogger from 'redux-logger';
 import rootReducer from '../reducers';
+import * as instrument from '../../shared/instrument';
+
+const instrumenter = store => next => action => {
```

Comment:
On Fri, Apr 8, 2016 at 3:51 PM, Jordan Santell <notifications@github.com>
wrote:

> In ui/browser/store/store.js
> <https://github.com/mozilla/tofino/pull/42#discussion_r59099690>:
>
> > @@ -13,6 +13,14 @@ specific language governing permissions and limitations under the License.
> >  import { applyMiddleware, createStore } from 'redux';
> >  import createLogger from 'redux-logger';
> >  import rootReducer from '../reducers';
> > +import * as instrument from '../../shared/instrument';
> > +
> > +const instrumenter = store => next => action => {
>
> It's a bit hard to look at, but is the recommended way for middleware FWIW
>

Yeah, straight out of the docs.  It's like 10000 spoons when all you want
is to curry transparently...